request/js/main.js

```diff
@@ -0,0 +1,109 @@
+$(function() {
+
+    // QUESTIONS (by default the form starts by asking whether the request is a referral or not)
+    // Will most likely need to be moved to a JSON file at some point in the near future
+    var referralQuestionList = [
+      {"key":"name", "value": "Their Name", "helper": "What's their name?"},
+      {"key":"referer_name", "value": "Your Name", "helper": "What's your name?"},
+      {"key":"age", "value": "Their Age", "helper": "How old are they?"},
+      {"key": "gender", "value": "Gender", "helper": "What gender do they identify as?"},
+      {"key": "school", "value": "School Name", "helper": "What school do they attend?"},
+      {"key": "contact", "value": "Their Contact", "helper": "What's the best way to reach them?"},
+      {"key": "referer_contact", "value": "Your Contact", "helper": "What's the best way to reach you?"},
+      {"key": "situation", "value": "Please Explain", "helper": "Please provide any additional information."}
+    ]
+    var individualQuestionList = [
+      {"key":"name", "value": "Your Name", "helper": "What's your name?"},
+      {"key":"age", "value": "Your Age", "helper": "How old are you?"},
+      {"key": "gender", "value": "Gender", "helper": "What gender do you identify as?"},
+      {"key": "school", "value": "School Name", "helper": "What school do you attend?"},
+      {"key": "contact", "value": "Your Contact", "helper": "What's the best way to reach you?"},
+      {"key": "situation", "value": "Please Explain", "helper": "What thoughts are you having?"} // definitely rephrase
+    ];
+
+
+    // initialize standard form variables on page load
+    var helper = $("#helper");
+    var mainInput = $("#mainField");
+    var questionList = individualQuestionList;
+    var placeholder = mainInput.attr("placeholder");
+    var inputJSON = {};
+    var currentQuestion = -1;
+    var referral = false;
+
+
+    // Process current question and pull up next question
+    function next() {
+      var input = mainInput.val(); // grab main textfield input
+
+      // decide whether the request is a referral or not
+      if (currentQuestion == -1 && (input.toLowerCase() == "yes" || input.toLowerCase() == "y")) {
```

Comment:
again, let's generalize this so that any question can introduce its own follow up questions